user-bag-ui/src/modules/client/src/components/manage-user/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {ChangeEvent} from 'react';
+import {UserModel} from "../../../../../models";
+import {Button, FormGroup, InputGroup} from "@blueprintjs/core";
+
+interface ManageUserProps {
+    user: UserModel;
+    onBack: () => void;
+}
+
+const ManageUser: React.FC<ManageUserProps> = ({user, onBack}) => {
```

Comment:
Looks quite similar to your `CreateUser` component, could there be some shared components to pull out?